docs/src/man/joins.md

```diff
@@ -353,21 +353,32 @@ case pass `matchmissing=:notequal`:
 ```jldoctest joins
 julia> innerjoin(df1, df2, on=:id, matchmissing=:notequal)
 1×3 DataFrame
- Row │ id      a      b     
+ Row │ id      a      b
      │ Int64?  Int64  Int64
 ─────┼──────────────────────
    1 │      1      1      1
 ```
 
+## Support for `-0.0` and `NaN` in joins
```

Comment:
We mention this already under the "Key value comparisons and floating point values" section. How about putting the example there?

Reply:
ah - right. I was looking for it, but missed. Moved.